pkg/controller/group/group_controller.go

```diff
@@ -192,3 +236,57 @@ func (c *Controller) deleteRoleBindings(group *iam1alpha2.Group) error {
 
 	return nil
 }
+
+func (c *Controller) multiClusterSync(group *iam1alpha2.Group) error {
+
+	obj, err := c.federatedGroupLister.Get(group.Name)
+	if obj == nil {
```

Comment:
Internal server error should be returned first, create the federation object if the error is `not found`.

Reply:
Refined.